index.js

```diff
@@ -392,12 +392,15 @@ function inject (bot) {
         const block = bot.blockAt(new Vec3(b.x, b.y, b.z), false)
         const tool = bot.pathfinder.bestHarvestTool(block)
         fullStop()
-        bot.equip(tool, 'hand', function () {
-          bot.dig(block, function (err) {
-            lastNodeTime = performance.now()
-            if (err) resetPath('dig_error')
-            digging = false
-          })
+        bot.equip(tool, 'hand').then(function () {
+          bot.dig(block)
+            .catch(_ignoreError => {
+              resetPath('dig_error')
```

Comment:
you need to do digging=false here or this is a behavior change

Reply:
I think it should work as the .then after it always triggers? The original code makes digging = false after dig ended with an error or not. So this should do the same? Or am I wrong.

Reply:
ah yes true